sympy/simplify/radsimp.py

```diff
@@ -1067,16 +1067,50 @@ def split_surds(expr):
 
 def _split_gcd(*a):
     """
-    split the list of integers ``a`` into a list of integers, ``a1`` having
-    ``g = gcd(a1)``, and a list ``a2`` whose elements are not divisible by
-    ``g``.  Returns ``g, a1, a2``
+    Parameters
+    =========
+
+    *a : integers, floats
```

Comment:
the earlier docstring explicitly stated `*a` to be list of integers. Even though it would work with floats, though the purpose of function of may not be that. Also since the function is an internal function, not a public one.